docgen/src/guides/calendar-widget.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Create a calendar widget
+mainTitle: Guides
+layout: main.pug
+name: customize
+category: guides
+withHeadings: true
+navWeight: 20
+editable: true
+githubSource: docgen/src/guides/calendar-widget.md
+---
+
+Search results often need to be refined by dates. Calendars are a good visual way to improve the user experience when it comes to filtering dates. InstantSearch provides custom widgets to create reusable components within your app.
+
+By the end of this guide, you'll understand how to implement a calendar widget to refine your results based on a single date and a range of dates. You can [preview](https://community.algolia.com/instantsearch.js/v2/examples/calendar-widget/) and [download](https://community.algolia.com/instantsearch.js/v2/examples/calendar-widget.zip) the concert app that you'll be able to build.
```

Comment:
Links should be relative.

Reply:
Right, should I make all links relative or only these two?
I've seen some other guides using absolute links.